### ISSUE TITLE
[Feature]: Exclude selected spaces from backup policy

### ISSUE THREAD

**Issue by Sclafus** [httpie/desktop#118]
Hello, thanks for this amazing application! It's truly amazing.
I have a work PC and a personal PC, both of them have HTTPie installed and logged in. 
I want to have my personal space available in my work PC, but I do not want my work space available in my personal PC due to legal reasons. In this case, it would be:
- Personal space, backed up on the cloud
- Work space, local only
Would it be possible to add this feature? Even a very small window with a toggle switch per-space would be enough!

**Comment by Sclafus** [comment]
Forgot to mention: the same concept should be applied to environments as well!

**Comment by jkbrzt** [comment]
> Forgot to mention: the same concept should be applied to environments as well!

We have that! It's called local-only environments:

https://httpie.io/blog/changelog-0017#local-only-environments

**Comment by Sclafus** [comment]
> https://httpie.io/blog/changelog-0017#local-only-environments

Yes, that's a very fair point, but you cannot change it after it's created :(

**Comment by jkbrzt** [comment]
> Yes, that's a very fair point, but you cannot change it after it's created :(

Yes, it’s immutable to avoid confusing sync scenarios. You can duplicate the environment as local-only and then delete the original one:

<img width="726" alt="Screenshot 2023-10-10 at 9 00 13 AM" src="https://github.com/httpie/desktop/assets/326885/7f1b7749-2a7b-4fa7-89e5-5d248e012663">
